index.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Angular Application</title>
+</head>
+<body>
+
+    <div>
+        <div ng-controller="MainController">
+            Hello, {{ name }}!
+        </div>
+    </div>
+
+    <script src="js/angular.js"></script>
+    <script src="js/angular-route.js"></script>
+    <script src="js/app/controllers/MainController.js"></script>
```

Comment:
since we're linking to the `MainController.js` file, might as well link to `app.js` here as well.